ecl/hql/hqlgram2.cpp

```diff
@@ -10600,7 +10600,7 @@ IHqlExpression * HqlGram::resolveImportModule(const attribute & errpos, IHqlExpr
     const char * parentName = str(parent->queryId());
     if (name == _container_Atom)
     {
-        const char * containerName = str(parent->queryFullContainerId());
+        const char * containerName = str(parent->queryBody()->queryFullContainerId());
```

Comment:
parent was a symbol annotation, rather than the CHqlRemoteScope, and the symbol annotation didn't have the pull path of the attribute as its container.  Calling queryBody() ensures the base expression is accessed.